UefiCpuPkg/PiSmmCpuDxeSmm/SmmProfile.c

```diff
@@ -840,12 +841,14 @@ InitSmmProfileInternal (
   //
   // Start SMM profile when SmmReadyToLock protocol is installed.
   //

```

Comment:
This comment is not right.
It can be changed to “Save the location of SMM Profile Data in UEFI variable for traditional SMM".